sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -815,6 +815,9 @@ def list_blobs(
         :keyword int results_per_page:
             Controls the maximum number of Blobs that will be included in each page of results if using
             `ItemPaged.by_page()`.
+        :keyword str start_from: Specifies the full path (inclusive) to list paths from.
+            For non-recursive list, only one entity level is supported.
+            For recursive list, multiple entity levels are supported.
```

Comment:
Since we are in the non-recursive function we probably don't need to mention the recursive restriction. In fact, for list_blobs, we probably don't need to mention it at all.

For `walk_blobs` (the recursive one) we should just mention "only one entity level is supported".